packages/components/psammead-most-read/src/item.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { node, oneOf, shape, string } from 'prop-types';
+import styled from 'styled-components';
+import { getFoolscap, getDoublePica } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  getSerifLight,
+  getSerifMedium,
+} from '@bbc/psammead-styles/font-styles';
+
+const ItemWrapper = styled.div``;
+
+export const StyledCountSpan = styled.span`
+  ${({ script }) => script && getFoolscap(script)};
+  ${({ service }) => getSerifLight(service)}
+  color: ${C_POSTBOX};
+  margin: 0; /* Reset */
+  padding-bottom: ${GEL_SPACING};
+  display: inline-block;
+  min-width: 3rem;
+  
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    width: 4rem;
+  }
+`;
+
+const StyledLink = styled.a`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSerifMedium(service)}
+  color: ${C_EBON};
+  text-decoration: none;
+  padding-bottom: ${GEL_SPACING};
+  
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
+export const MostReadItem = ({
+  dir,
+  lastUpdated,
+  script,
+  service,
+  item: { title, href },
+}) => (
+  <ItemWrapper dir={dir}>
+    <StyledLink href={href} script={script} service={service}>
+      {title}
+    </StyledLink>
+    {lastUpdated}
+  </ItemWrapper>
+);
+
+StyledCountSpan.propTypes = {
+  service: string.isRequired,
+  script: shape(scriptPropType).isRequired,
+};
+
+MostReadItem.propTypes = {
+  dir: string,
```

Comment:
Do we really need `dir` prop? I can see it is being used on the stories, but removing it and changing the service from the service knob to a `rlt` service would achieve the same result.

Reply:
I think it would be useful to have this prop being explicitly declared even if it's being passed automatically in storybook.

Reply:
In most of our components in `psammead` we pass in `dir` either for correct padding or floats. `dir` in `MostReadItem` doesn't require any of those.

Reply:
@sadickisaac In this case, `dir` isn't for layout - it is for correctly stating the direction of the text contained within the MostReadItem. It is an element attribute needed for the HTML output, not for the CSS.